packages/grammy-nestjs/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@grammyjs/nestjs",
   "description": "grammY module for NestJS",
-  "version": "0.3.1",
+  "version": "0.3.2",
```

Comment:
This is changed automatically with the release script.  I can walk you thru this

Reply:
It would be awesome if you two could coordinate! I myself didn't dig into the tooling of this repo too much yet, so I'm unfortunately not much of a help …

Reply:
Got the husky pre-lint stuff working. Merging this PR.